src/client/SignIn.ts

```diff
@@ -26,7 +26,7 @@ export async function SignIn(username?: string, password?: string): Promise<Big>
         const onSuccess = (): void => resolve(new Big(200));
         const onError = (error: { status: number }): void => resolve(new Big(error.status));
 
-        mx.login(username, password, onSuccess, onError as any);
+        mx.login2(username, password, Boolean(useAuthToken), onSuccess, onError as any);
```

Comment:
This means that offline apps have to set this argument as true to be able to work as previously using an authentication token.

Reply:
Added a fallback to the old login method